src/pages/TripBookingPage/PassengerDetailsForm/validation.ts

```diff
@@ -0,0 +1,27 @@
+import * as Yup from 'yup';
+import { TFunction } from 'i18next';
+
+const getValidationSchema = (t: TFunction<'passengerDetails'>) =>
+    Yup.object().shape({
+        passengers: Yup.array().of(
+            Yup.object().shape({
+                fullName: Yup.string()
+                    .required(t('fullNameRequired'))
+                    .test('valid-fullName', t('fullNameLength'), (value) => {
```

Comment:
Full name validation fails when we enter blank space

Reply:
corrected.